### DISCUSSION TITLE
Amazing but Little Annoying

### DISCUSSION THREAD

**Discussion by HimanshuBari21** [community/community#20492] [Copilot Conversations]
As it is also a AI so yes it is little annoying, remembder when google assistant was very new, Its suggestions also annoys us.
and We be like "oh man let me type first! " something similar experience I got on this Github Copilot also.

But overall Its amazing in CSS and JavaScript (ReactJS, NodeJS, MongoDB).

using it from 3 Month.

So that my say ,would like to read about how other developers feels about it especially CSS Developers, did you feel useless guys?

thanks for reading...

**Comment by jfullstackdev** [comment]
just want to share this 

https://betterprogramming.pub/the-current-state-of-github-copilot-is-it-awesome-or-rubbish-bcede89069ef

**Comment by Dhiraj275** [comment]
But it will take a very long time to copilot to learn to make a whole website with working logic and DBMS
